.github/workflows/nightly.yaml

```diff
@@ -1,45 +1,34 @@
 name: Nightly build
-
 on:
   schedule:
     - cron: "0 3 * * *"
   push:
     branches:
       - main
+  pull_request:
```

Comment:
Are we supposed to create a `nightly` image on pull-requets or is it a leftover?

Reply:
My bad. That was left over from testing...